lib/Slackbot_worker.js

```diff
@@ -105,6 +111,9 @@ module.exports = function(botkit, config) {
             });
 
             bot.rtm.on('close', function() {
+                if (pingIntervalId) {
```

Comment:
I suppose not, just being overly defensive in case something weird happened in the open handler and the interval didn't get setup properly.  I can remove it if you think it is useless